### DISCUSSION TITLE
Version 2.1.1

### DISCUSSION THREAD

**Discussion by knot126** [knot126/Smash-Hit-Blender-Tools#40] [Announcements]
* Fixes #39: particles will now work!
* Fixes a bug when exporting a visible box which previously used a template

<hr /><em>This discussion was created from the release <a href='https://github.com/Smashing-Tech/Smash-Hit-Blender-Tools/releases/tag/2.1.1'>Version 2.1.1</a>.</em>

**Comment by OverloadedTech** [comment]
I'm glad you fixed it
